src/components/interface/TableHeader.jsx

```diff
@@ -1,32 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import SearchIcon from '@material-ui/icons/Search';
-import Input from '@material-ui/core/Input';
 import TableCell from '@material-ui/core/TableCell';
 import TableHead from '@material-ui/core/TableHead';
 import TableRow from '@material-ui/core/TableRow';
 import { withStyles } from '@material-ui/core/styles';
 import styles from '../styles';
-import Filter from './filter/index';
 
-
-const TableHeader = ({ headings, classes }) => (
+const TableHeader = ({ headings }) => (
   <TableHead>
-    <div style={{ display: 'flex', margin: '5px' }}>
```

Comment:
Just to confirm, this is the DOM object you are moving out into a different class correct?